Src/BootCamp.Chapter/InvalidBalancesException.cs

```diff
@@ -4,5 +4,18 @@ namespace BootCamp.Chapter
 {
     public class InvalidBalancesException : Exception
     {
+        private string _Msg;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_Msg' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)